integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/util/IpvCoreStubUtil.java

```diff
@@ -111,4 +115,23 @@ public static HttpResponse<String> sendSessionRequest(String apiPath, String ses
                         .build();
         return sendHttpRequest(request);
     }
+
+    public static HttpResponse<String> sendAuthorizationRequest(
```

Comment:
Change the signature of this method to the following:
sendAuthorizationRequest(String apiPath, String sessionId, String redirectUri, String clientId)
& create the `requestParameters` list inside this method to avoid unnecessary duplication of code in the `APISteps` class